projects/kubernetes/kubernetes/Makefile

```diff
@@ -50,7 +50,6 @@ pause:
 	mkdir -p $(PAUSE_DST_DIR)
 	# Older versions have the pause.c file stored under kubernetes/build/pause/
 	test -f $(PAUSE_SRC_DIR)/linux/pause.c && cp $(PAUSE_SRC_DIR)/linux/pause.c $(PAUSE_DST_DIR) || cp $(PAUSE_SRC_DIR)/pause.c $(PAUSE_DST_DIR)
-	rm -rf ./kubernetes
```

Comment:
What was the issue/motivation here?

Reply:
That gets used in a step later on and it is missing.  It is something that should probably be done, but not here.

Reply:
I'm going to move this change to another PR. I assume that was put there because of the space issue for tests and I'll come up with some other fix for that.